.github/scripts/hermetic_library_generation.sh

```diff
@@ -99,22 +94,18 @@ docker run \
   -v "$(pwd):${workspace_name}" \
   -v "$HOME"/.m2:/home/.m2 \
   -e GENERATOR_VERSION="${generator_version}" \
-  gcr.io/cloud-devrel-public-resources/java-library-generation:"${image_tag}" \
-  --baseline-generation-config-path="${workspace_name}/${baseline_generation_config}" \
-  --current-generation-config-path="${workspace_name}/${generation_config}"
+  gcr.io/cloud-devrel-public-resources/java-library-generation:"${image_tag}"
```

Comment:
Is the commit history going to be populated correctly if we don't pass the baseline generation config into the image?

Reply:
No, the commit history has to go through baseline/current config comparison.

However, in this repo, we don't have config changes other than googleapis commit since everything else is baked in the image.